app/views/hyrax/homepage/_featured_fields.html.erb

```diff
@@ -1,16 +1,14 @@
-<h3>
-  <span class="sr-only">Title</span>
-  <%= link_to truncate(featured.title.first, length: 28, separator: ' '), [main_app, featured] %>
-</h3>
 <div>
-  <span class="sr-only">Depositor</span>
-  <%= link_to_profile featured.depositor("no depositor value") %>
-</div>
-<div>
-  <span class="sr-only">File type</span>
-  <%= link_to_facet_list(featured.resource_type, 'resource_type', 'no resource specified') %>
-</div>
-<div>
-  <span class="sr-only">Keywords</span>
-  <%= link_to_facet_list(featured.keyword, 'keyword', 'no keywords specified') %>
+  <div class="featured-item-title">
```

Comment:
How would you feel about including the bare strings in this file in i18n so they can be translated in the next release? (These strings may in fact already be translated...)

Reply:
WFM. Will update.

Reply:
Done